src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier{
```

Comment:
Extending ColorSupplier is not necessary if you're only calling the superclass method without any changes. You can create an instance of ColorSupplier and use its method directly.